src/pages/login.tsx

```diff
@@ -0,0 +1,19 @@
+import Head from 'next/head';
+import { FC } from 'react';
+
+import LoginForm from '@components/molecules/LoginForm';
+import AuthLayout from '@src/layouts/auth';
+
+const Login: FC = () => {
+  return (
+    <AuthLayout>
+      <Head>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
I'd move this up to `_document.tsx`

Reply:
Next.js warns against using viewport meta tags in `_document.tsx`

Reply:
Ah fairs, didn't know that - [they suggest `_app.tsx` instead](https://nextjs.org/docs/messages/no-document-viewport-meta#:~:text=The%20viewport%20tag%20should%20be%20handled%20by%20next/head%20in%20pages/_app.js)